### PR TITLE
Fix #2075: Increase node.js heap from 1.5 to 2 GB.

### LINKED ISSUE #2075: Out of memory in toolsJS/test virtually every time
Possible track to fix it: http://stackoverflow.com/questions/26094420/fatal-error-call-and-retry-last-allocation-failed-process-out-of-memory

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

**Comment by nicolasstucki** [comment]
Review by @sjrd

**Comment by sjrd** [comment]
We should do this only in our CI for toolsJS/test. Or only in the settings for toolsJS. Definitely not as a default for all `NodeJSEnv`s in the world.

**Comment by sjrd** [comment]
LGTM

**Comment by sjrd** [comment]
Don't merge, it succeeded by chance!

**Comment by sjrd** [comment]
There was still this error in one of the jobs:
```

<--- Last few GCs --->

  817148 ms: Scavenge 1394.5 (1455.7) -> 1394.5 (1455.7) MB, 1.2 / 0 ms (+ 11.5 ms in 2 steps since last GC) [allocation failure] [incremental marking delaying mark-sweep].
  820443 ms: Mark-sweep 1394.5 (1455.7) -> 1391.3 (1455.7) MB, 3293.7 / 0 ms (+ 14.3 ms in 4 steps since start of marking, biggest step 11.2 ms) [last resort gc].
  823692 ms: Mark-sweep 1391.3 (1455.7) -> 1394.1 (1455.7) MB, 3249.2 / 0 ms [last resort gc].


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x26d4c737399 <JS Object>
    2: $$plus__T2__T2__sc_Seq__sci_HashMap [/localhome/jenkins/a/workspace/scalajs-task-worker/tools/js/target/scala-2.11/scalajs-tools-test-fastopt.js:361252] [pc=0x290b32ab1e1f] (this=0x1de0c79ecbb9 <a $c_sci_HashMap with map 0x370741c2ddf1>,elem1=0x1de0c79ecb91 <a $c_T2 with map 0x370741c1a8e9>,elem2=0x1de0c79ecb69 <a $c_T2 with map 0x370741c1a8e9>,elems=0x1de0c79ecb19 <a $c_sjs_js_WrappedArr...

[error] FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
```
Apparently the option does have any effect. The GC is still stuck at around 1.5 GB.

File: js-envs/src/main/scala/org/scalajs/jsenv/ExternalJSEnv.scala [hunk 1]
```diff
@@ -107,6 +107,8 @@ abstract class ExternalJSEnv(
       for ((name, value) <- vmEnv)
         pBuilder.environment().put(name, value)
 
+      logger.info("Starting process: " + allArgs.mkString(" "))
```
Comment (sjrd) [inline js-envs/src/main/scala/org/scalajs/jsenv/ExternalJSEnv.scala]: This should be `debug`

File: js-envs/src/main/scala/org/scalajs/jsenv/ExternalJSEnv.scala [hunk 2]
```diff
@@ -107,6 +107,8 @@ abstract class ExternalJSEnv(
       for ((name, value) <- vmEnv)
         pBuilder.environment().put(name, value)
 
+      logger.debug("Starting process: " + allArgs.mkString(" "))
+
```
Comment (gzm0) [inline js-envs/src/main/scala/org/scalajs/jsenv/ExternalJSEnv.scala]: Could you put this in a separate commit with a separate commit message? (It just might be a bit confusing after the fact that "increasing the Node.js heap" changes what the envs log).

**Comment by sjrd** [comment]
LGTM

**Pull request merged**